server/src/main/java/com/linecorp/centraldogma/server/internal/mirror/AbstractMirror.java

```diff
@@ -54,12 +54,14 @@
     private final URI remoteRepoUri;
     private final String remotePath;
     private final String remoteBranch;
+    private final String gitIgnore;
     private final ExecutionTime executionTime;
     private final long jitterMillis;
 
     protected AbstractMirror(Cron schedule, MirrorDirection direction, MirrorCredential credential,
                              Repository localRepo, String localPath,
-                             URI remoteRepoUri, String remotePath, @Nullable String remoteBranch) {
+                             URI remoteRepoUri, String remotePath, @Nullable String remoteBranch,
+                             @Nullable String gitIgnore) {
```

Comment:
How about accepting an array of String which is `Iterable<String> gitIgnore` instead of just a String?
People usually use multiple lines to exclude the files. >_<

Reply:
Had a chat with @yyuunn0044 and it's fine as it is. 😄 
We are eventually going to implement UI for mirroring setup so it's better to take a String instead of an array of Strings. 😄